www/index.php

```diff
@@ -14,9 +14,14 @@
 
 $config = Configuration::get_configuration();
 $app = new Slim();
+$app->config('debug', true);
```

Comment:
should we wrap this into something like `if (getenv('MORGUE_ENVIRONMENT') === "development")`?